validate.cwl

```diff
@@ -4,60 +4,58 @@
 #
 cwlVersion: v1.0
 class: CommandLineTool
-baseCommand: python
+baseCommand: [python3, /validate.py]
 
 hints:
   DockerRequirement:
-    dockerPull: python:3.8.8-slim-buster
+    dockerPull: docker.synapse.org/syn26720921/scoring:v1
 
 inputs:
-  - id: input_file
+  - id: submission_file
     type: File?
   - id: entity_type
     type: string
+  - id: input_files
+    type: File[]?
+  - id: condition
```

Comment:
`validate.py` mentions that these inputs are required - should they be optional here?